app/web/src/features/auth/signup/BasicForm.tsx

```diff
@@ -42,7 +42,11 @@ export default function BasicForm() {
   const mutation = useMutation<void, GrpcError, SignupBasicInputs>(
     async (data) => {
       const sanitizedEmail = lowercaseAndTrimField(data.email);
-      const state = await service.auth.startSignup(data.name, sanitizedEmail);
+      const sanitizedName = data.name.trim();
```

Comment:
```suggestion
      const sanitizedName = lowercaseAndTrimField(data.name);
```
Might be worth using the same function for sanitising the name as the one used for email? I believe it's also case sensitive.

Reply:
We probably don't want to lowercase people's names lol

Reply:
Oh wait I thought this is the user name oops, yeah probably fine to ignore this then. Doing this kinda stuff on the frontend low key bugs me as I think the backend should really handle it tbh... 🙄